pages/technical/identifiers.rst

```diff
@@ -43,14 +44,22 @@ AGBCode
 
     urn:oid:2.16.840.1.113883.2.4.6.1:00000007
 
+Hospital OID
+............
+In this case the hospital has its own OID.
+::
+
+    urn:ietf:rfc:3986:2.16.840.1.113883.2.4.3.55
+
 X500Name
 ........
 ::
 
     urn:ietf:rfc:1779:O=Nuts, OU=Healthcare, C=NL, ST=Gelderland, L=Eibergen, CN=nuts_corda_development_local
 
-Endpoint
+Consent Endpoint
 ........
+Endpoint to connect with to synchronise consents
 ::
 
-    urn:nuts:endpoint:consent
+    urn:oid:1.3.6.1.4.1.54851.2:consent
```

Comment:
or `urn:oid:1.3.6.1.4.1.54851.2.1`?

Reply:
Lets use the `urn:system:value` notation. In that case it would become
`urn:ietf:rfc:1.3.6.1.4.1.54851.2.1`. 
You suggestion means keeping tables of approved values. That would suggest Nuts will be responsible for approving endpoint types and thus use-cases. This can be good or bad. What do you think?

Reply:
indeed, thats the argument that kills my suggestion.